config/authorization/config.ex

```diff
@@ -592,8 +592,31 @@ defmodule Acl.UserGroups.Config do
       },
 
       %GroupSpec{
-        name: "parliament-flow",
-        useage: [:read, :write, :read_for_write],
+        name: "parliament-flow-read",
+        useage: [:read],
+        access: access_by_role(
+          admin_roles()
+          ++ secretarie_roles()
+          ++ ovrb_roles()
+          ++ kort_bestek_roles()
+          ++ minister_roles()
+          ++ kabinet_dossierbeheerder_roles()
+          ++ kabinet_medewerker_roles()
+          ++ overheid_roles()
+        ),
+        graphs: [
+          %GraphSpec{
+            graph: "http://mu.semte.ch/graphs/system/parliament",
+            constraint: %ResourceConstraint{
+              resource_types: parliament_resource_types()
+            }
+          }
+        ]
+      },
+
+      %GroupSpec{
+        name: "parliament-flow-write",
+        useage: [:write, :read_for_write],
         access: access_by_role(
           admin_roles()
           ++ secretarie_roles()
```

Comment:
are `secretarie_roles()` and `kort_bestek_roles()` really needed here? I was under the impression that they would never send a case to the parliament.

Reply:
Replying here for all the comments about permission roles:

Ticket specified admin and kabinetdossierbeheerder roles. Because we often use "admin" and secretarie interchangeably, I assumed it was the case here as well (especially given that on PROD there are only 14 admin profiles, 9 of which are us developers so the admin users who will actually use this feature are very very limited).

Will change to strictly admin and kabinetdossierbeheerder.

Reply:
Should `minister_roles` also be removed here?

Reply:
as discussed in standup: admin, minister and kabinetsdossierbeheerder can stay